autoload/neomake/makers/rust.vim

```diff
@@ -0,0 +1,19 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#rust#EnabledMakers()
+    return ['rustc']
+endfunction
+
+function! neomake#makers#rust#rustc()
+    return {
+        \ 'args': ['-o /dev/null'],
```

Comment:
Each arg should take its own entry in the list. Also, /dev/null is env specific, so we have a utility for that. Here is a corrected version:

```
\ 'args': ['-o', neomake#utils#DevNull()],
```